examples/pipelines/fl_cross_silo_literal/submit.py

```diff
@@ -51,9 +51,9 @@
 parser.add_argument(
     "--example",
     required=False,
-    choices=["MNIST", "HELLOWORLD"],
+    choices=["MNIST", "HELLOWORLD", "NER"],
```

Comment:
I believe we do not want to alter files not related to example

Reply:
I'm not shocked by this. If the generic examples also work, we might as well have them do the NER task. **IF** that's happening though, why only with literal and not native?

Reply:
my bad! The literal/factory `submit.py` is pretty much the same for the `NER` example but the `config.yml` is different. So I need to remove this from here.